Matlab_Scripts/5_Depth_Colourless/Study/markerinfo.txt

```diff
@@ -0,0 +1,16 @@
+Sent By Unity	Picked up by Vamp	 Event		Recoded as
```

Comment:
lets not replicate these into the study folder until we agree they are done in the pilot